lib/main.coffee

```diff
@@ -1,114 +1,97 @@
-{CompositeDisposable} = require 'atom'
+{readCsonFile} = require './promise-helper'
 path = require 'path'
-promiseHelper = require './promise-helper'
+
+grammarRE = /\/language-[^\/]+\/(grammars|settings)\/[^\/]+\.(?:c|j)son/
```

Comment:
This must be set in the package configuration.

Also I think will not work on Windows due to the Windows path separator `\`.

Reply:
What exactly do you think should be configurable?